### PR TITLE
Added Swedish translation

### PR DESCRIPTION
**Pull request by PontusHorn**

### CODE REVIEW

**Comment by remy** [comment]
Isn't swedish .se? The file should match the language code, and we'll need to get a +1 from another Swedish speaker - otherwise thank you!

**Comment by coffeepunk** [comment]
+1

Country code: se, language code: sv

File: index-sv.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+.footer p {
+  text-align: left;
+}
+</style>
+</head>
+<body>
+<h1>Uppförandekod för konferens</h1>
+
+<p>Alla deltagare, talare, sponsorer och volontärer på vår konferens är skyldiga att acceptera följande uppförandekod. Arrangörerna kommer att upprätthålla dessa riktlinjer under hela evenemanget. Vi väntar oss samarbete från alla deltagare för att bidra till att säkerställa en säker miljö för alla.</p>
```
Comment (parse) [inline index-sv.html]: Suggestion:
<p>Alla deltagare, talare, sponsorer och volontärer på vår konferens är skyldiga att acceptera följande uppförandekod. Arrangörerna kommer att upprätthålla dessa riktlinjer under hela evenemanget. Vi förväntar oss samarbete från alla deltagare för att säkerställa en säker miljö för alla.</p>

File: index-sv.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+<h2>Den mindre snabba versionen</h2>
+
+<p>Trakasserier inkluderar offensiva verbala kommentarer med anknytning till kön, könsidentitet och -uttryck, ålder, sexuell läggning, funktionshinder, utseende, kroppsstorlek, ras, religion, sexuella bilder på offentliga platser, avsiktlig skrämsel, stalking, förföljelse, trakasserande fotografering eller inspelning, ihållande avbrott i föredrag eller andra aktiviteter, olämplig fysisk kontakt, och ovälkommen sexuell uppmärksamhet.</p>
+
+<p>Deltagare som ombeds att upphöra med trakasserande beteende förväntas göra detta omedelbart.</p>
+
+<p>Sponsorer är också föremål för antitrakasseri-policyn. Framför allt ska sponsorer inte använda sexualiserade bilder, aktiviteter, eller annat material. Monterpersonal (inklusive volontärer) bör inte använda sexualiserade kläder/uniformer/kostymer, eller på annat sätt skapa en sexualiserad miljö.</p>
+
+<p>Om en deltagare bedriver trakasserier kan konferensarrangörerna vidta de åtgärder de anser vara lämpliga. Det kan innebära en varning eller utvisning från konferensen utan återbetalning.</p>
```
Comment (parse) [inline index-sv.html]: Suggestion:
<p>Om en deltagare trakasserar andra kan konferensarrangörerna vidta de åtgärder de anser vara lämpliga. Det kan innebära en varning eller utvisning från konferensen utan återbetalning.</p>

**Comment by PontusHorn** [comment]
Fair suggestions. They've been added.

**Comment by remy** [comment]
@coffeepunk - thanks for the clarification.

So, based on how the URL structure works, since the URL will be <code>.confcodeofconduct.com, I *think* it should be `se.html`.

@PontusHorn thoughts?

**Comment by ralphtheninja** [comment]
:+1: Flawless Swedish :)

**Comment by PontusHorn** [comment]
I think sv-index.html makes more sense as the other languages seem to go by [ISO 639-2] (http://www.loc.gov/standards/iso639-2/php/code_list.php), except Traditional Chinese. I'm not sure what kind of code tw is. The Brazilian Portuguese is also using the full locale code, which is probably preferable for all of them if you want something really future proof ;) That'd be sv-se in this case.

**Comment by gummesson** [comment]
I'll take a quick (but nitpicky) look through this. :)

**Comment by PontusHorn** [comment]
Great suggestions, thanks!

**Comment by remy** [comment]
All good?

**Comment by PontusHorn** [comment]
:+1:

**Comment by gummesson** [comment]
Looks good to me. :+1:

**Comment by ralphtheninja** [comment]
My swedish is rusty ;)

**Pull request merged**